simhalt.cc

```diff
@@ -2610,6 +2610,46 @@ void haltestelle_t::get_short_freight_info(cbuffer_t & buf) const
 }
 
 
+void haltestelle_t::get_transfers_info(cbuffer_t& buf) const
+{
+	int idx = get_finance_history( 1, HALT_ARRIVED) == 0 ||
+		get_finance_history( 1, HALT_DEPARTED) == 0 ? 0 : 1;
+	auto transfers = get_finance_history( idx, HALT_ARRIVED) <
+		get_finance_history( idx, HALT_DEPARTED) ?
+		get_finance_history( idx, HALT_ARRIVED) :
+		get_finance_history( idx, HALT_DEPARTED);
+	buf.printf("%d ", transfers);
+	buf.append(translator::translate("transfers"));
+	buf.append("\n");
+}
+
+void haltestelle_t::get_percent_info(cbuffer_t& buf) const
+{
+	bool got_one = false;
+	sint64 sum = 0;
+	for(unsigned int i=0; i<goods_manager_t::get_count(); i++) {
+		const goods_desc_t *wtyp = goods_manager_t::get_info(i);
+		if(gibt_ab(wtyp)) {
+			sum += get_capacity( i>2?2:i );
+		}
+	}
+	if (sum > 0) {
+		got_one = true;
+	}
+
+	if(got_one) {
```

Comment:
(nit) Checking `sum > 0` seems to be enough.

Reply:
this might need to change depending on de decision for: https://github.com/teamhimeh/simutrans/pull/55/files#r1774599085